### PR TITLE
py-ompcb-merge: new package

### PR DESCRIPTION
**Pull request by VitoriaDMP**
@hyviquel

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the home page and version sha256.

**Comment by tldahlgren** [comment]
Adding @adamjstewart since he's the python package guru/maintainer.

File: var/spack/repos/builtin/packages/py-ompcb-merge/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+from spack.package import *
+
+
+class PyOmpcbMerge(PythonPackage):
+    """Command-line tool to merge trace timelines from OMPC runtime."""
+
+    homepage = "https://gitlab.com/ompcluster/ompcbench"
+
+    url = "https://github.com/VitoriaDMP/ompcb-merge/raw/master/ompcb-merge-1.0.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ompcb-merge/package.py:14]: I would rather use the official download source than a random file on GitHub.

File: var/spack/repos/builtin/packages/py-ompcb-merge/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyOmpcbMerge(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ompcb-merge/package.py:9]: Why is it called ompcb-merge and not ompcbench?

File: var/spack/repos/builtin/packages/py-ompcb-merge/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    homepage = "https://gitlab.com/ompcluster/ompcbench"
+
+    url = "https://github.com/VitoriaDMP/ompcb-merge/raw/master/ompcb-merge-1.0.tar.gz"
+
+    maintainers("VitoriaDMP")
+
+    version("1.0", sha256="807a311e93fe5063058194a75612dbabc26dfcde19adef33ba71fa72191c8d39")
+
+    depends_on("py-orjson@3.6:", type=("build"))
+    depends_on("py-click@8.0:", type=("build"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-ompcb-merge/package.py:21]: Seems to be missing a lot of deps

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**